admin/XoxzoListTableSms.php

```diff
@@ -213,26 +213,46 @@ public function process_bulk_action() {
                 exit(0);
             }
 
-            $curl_arr = array();
-            $master = curl_multi_init();
-            for($i = 0; $i < $node_count; $i++)  {
-                $url =$urls[$i];
-                $curl_arr[$i] = curl_init($url);
-                curl_setopt($curl_arr[$i], CURLOPT_RETURNTRANSFER, true);
-                curl_setopt($curl_arr[$i], CURLOPT_USERPWD, get_option("woocommerce_xoxzo_sid").":".get_option("woocommerce_xoxzo_auth_token"));
-                curl_setopt($curl_arr[$i], CURLOPT_HTTPAUTH, CURLAUTH_BASIC);
-                curl_multi_add_handle($master, $curl_arr[$i]);
-            }
 
-            do {
-                curl_multi_exec($master,$running);
-            }
-            while($running > 0);
+                
+        $encodedAuth = base64_encode(get_option("woocommerce_xoxzo_sid").":".get_option("woocommerce_xoxzo_auth_token"));
 
-            $results = [];
-            for($i = 0; $i < $node_count; $i++)  {
-                $results[] = curl_multi_getcontent  ( $curl_arr[$i]  );
-            }
+        $response = wp_remote_request( $url,
+            array(
+                "headers" => array(
+                    "Authorization" => "Basic ".$encodedAuth
+                ),
+                "method" => "GET",
+            )
+        );
+
+    
+        $results = json_decode($response['body'], true);
+
+       
+
+
+
+            // $curl_arr = array();

```

Comment:
Can you remove the commented codes from the file? 
Comments should be for any explanation on what the code does or required arguments if it appears to be too obscure.